README.md

```diff
@@ -45,14 +50,16 @@ The top level of the configuration file looks like this:
       },
       "scheme": "https",
       "__snapName": "myUniqueSnapName",
-      "__snapType": "__GET"
+      "__snapType": "__REST""
     },
   ...
   ]
 }
 ```
 * The `snaps` entry needs to contain the `__snapName` key to uniquely identify the request. This name will be used to create a file with the snap result on disk.
-* `__snapType` specifies the type of request Kombucha is going to issue. `__GET` is a simple HTTP `GET` request, and the remaining parameters are interpreted accordingly: `queryItems` are converted into key-value pairs and appended onto the `path`, etc.
+* It is possible to specify any `HTTP` header field with `httpHeaders`  key, the value of a header needs to be a string. This key is requeried. Use `"httpHeaders": {}` to specify no headers.
+* The `body` key (not required) accepts arbitrary `JSON` and it will be used to set the body if the `HTTP` request.
+* `__snapType` specifies the type of request Kombucha is going to issue. `__REST"` is a simple HTTP `GET`, `POST`, `PUT`, `DELETE` or `PATCH` request (see the `httpMethod` key above), and the remaining parameters are interpreted accordingly: `queryItems` are converted into key-value pairs and appended onto the `path`, etc.
```

Comment:
I think you need to remove the extra quote inside of `__REST"`, seems like it is making the markdown parser mad!

Reply:
It worked :)